src/main/java/com/autotune/database/table/ExperimentDetailTable.java

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.database.table;
+
+import com.autotune.analyzer.utils.AnalyzerConstants;
+import com.fasterxml.jackson.databind.JsonNode;
+import jakarta.persistence.*;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+/**
+ * This is a Java class named ExperimentDetail annotated with JPA annotations.
+ * It represents a table named experiment_detail in a relational database.
+ * <p>
+ * The class has the following fields:
+ * <p>
+ * id: A unique identifier for each experiment detail.
+ * version: A string representing the version of the experiment.
+ * experimentName: A string representing the name of the experiment.
+ * clusterName: A string representing the name of the cluster.
+ * mode: A string representing the mode of the experiment.
+ * targetCluster: A string representing the target cluster for the experiment.
+ * performance_profile: A string representing the performance profile for the experiment.
+ * status: An enum representing the status of the experiment, defined in AnalyzerConstants.ExperimentStatus.
+ * extended_data: A JSON object representing extended data for the experiment.
+ * meta_data: A string representing metadata for the experiment.
+ * The ExperimentDetail class also has getters and setters for all its fields.
+ */
+@Entity
+@Table(name = "experiment_detail")
+public class ExperimentDetailTable {
```

Comment:
This is just a table entry, not the table itself, so it needs to be named more appropriately (Eg. KruizeExperimentEntry). Also the table name should be `kruize_experiment`